tests/test_models.py

```diff
@@ -0,0 +1,76 @@
+from django.contrib.auth import get_user_model
+from django.test import TestCase
+from django.urls import reverse
+
+from taxi.models import Manufacturer, Car
+
+
+class ManufacturerModelTests(TestCase):
+    def setUp(self) -> None:
+        self.manufacturer = Manufacturer.objects.create(
+            name="Test Manufacturer",
+            country="Test Country"
+        )
+
+    def test_str(self) -> None:
+        self.assertEqual(
+            str(self.manufacturer),
+            f"{self.manufacturer.name} "
+            f"{self.manufacturer.country}"
+        )
+
+
+class DriverModelTests(TestCase):
+    def setUp(self) -> None:
+        self.user_data = {
+            "username": "Test Username",
+            "first_name": "First",
+            "last_name": "Last",
+            "password": "test_password123",
+            "license_number": "ABC12345"
+
+        }
+        self.driver = get_user_model().objects.create_user(
+            **self.user_data
+        )
+
+    def test_str(self) -> None:
+        self.assertEqual(
+            str(self.driver),
+            f"{self.driver.username} "
+            f"({self.driver.first_name} {self.driver.last_name})"
```

Comment:
The format string for the driver's string representation should not include a space between the username and the parentheses. It should be formatted as `{self.driver.username} ({self.driver.first_name} {self.driver.last_name})` without the space after the username.